### DISCUSSION TITLE
Why you need a multisite and how to build one

### DISCUSSION THREAD

**Discussion by oglekler** [WordPress/developer-blog-content#118] [Topic Ideas]
The idea is to show what is possible with multisite, how themes such as [Stacks](https://wordpress.org/themes/stacks/) and plugins such as [Web Stories](https://wordpress.org/plugins/web-stories/) can create unique user experience and make site owner flexible with information presentation and creative with content beyond usual blocks and patterns.

I believe that when this topic writer will be found, the author will correct the scope and the title to match what he/she/they can actually write.

**Comment by bph** [comment]
From the [meeting notes:](https://make.wordpress.org/core/2023/06/02/developer-blog-editorial-meeting-01-june-2023/) "Regarding the first of these, it was considered that the topic is not yet defined enough. 

It also mentions using a third-party theme and a third-party plugin which prompted a discussion on whether third-party themes and plugins should be promoted in the Developer Blog. 

The consensus on this was that they should not be promoted in this way."


@OlaIola How would you narrow down the scope for this topic idea?

**Comment by bph** [comment]
Resources available on Learn WordPress for inspiration: 

- [Introduction to WordPress Multisite Networks](https://learn.wordpress.org/tutorial/introduction-to-wordpress-multisite-networks/)
- [Managing a WordPress Multisite Network](https://learn.wordpress.org/tutorial/managing-a-wordpress-multisite-network/)
- [Advanced Multisite Management](https://learn.wordpress.org/tutorial/advanced-multisite-management/)

Also quite relevant: [Developing for Multisite](https://learn.wordpress.org/tutorial/developing-for-multisite/)

**Comment by bph** [comment]
As the discussion didn't progress for a modified topic, I close it.
